src/main/antora/modules/ROOT/pages/jdbc/template.adoc

```diff
@@ -0,0 +1,48 @@
+[[jdbc.template]]
+= Template API
+
+Spring Data JDBC offers the javadoc:org.springframework.data.jdbc.core.JdbcAggregateTemplate[] as a more direct means to load and persist entities in a relational database.
```

Comment:
This page adds another variant of documenting Template API's.

I strongly suggest aligning with https://docs.spring.io/spring-data/relational/reference/r2dbc/entity-persistence.html to not introduce yet another variant of Template documentation.

Overall, `entity-persistence.html` would be a good place to explain the Template API as it fits pretty well in there surrounded by Optimistic Locking and ID generation.

Furthermore, the documentation misses update and delete method documentation.